packages/builder/src/pages/builder/app/[application]/settings/scripts.svelte

```diff
@@ -144,6 +144,17 @@
         minHeight={200}
         placeholder="&lt;script&gt;...&lt;/script&gt;"
       />
+      <Label size="L">
+        CSP whitelist<br />
+        <Link href="https://docs.budibase.com/docs/app-scripts" target="_blank">
```

Comment:
Is this link correct?

Reply:
Nope it's just a placeholder - I have a TODO in the description above just saying I need to get a proper docs link 👍